core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -80,7 +80,7 @@ func sendTransaction(gethClientWrapper strpkg.GethClientWrapper, a models.EthTxA
 	})
 
 	gasPriceGwei := fmt.Sprintf("%.2f", float64(a.GasPrice.ToInt().Int64())/1000000000)
-	logger.Debugf("BulletproofTxManager: Attempting transaction (eth_tx_attempt.id: %v): 0x%x at gas price %s Gwei", a.ID, signedTx.Hash(), gasPriceGwei)
+	logger.Debugf("BulletproofTxManager: Broadcasting transaction (eth_tx_attempt.id: %v): 0x%x at gas price %s Gwei", a.ID, signedTx.Hash(), gasPriceGwei)
```

Comment:
Would prefer not to have such gigantic output in log lines, much clearer to use `Debugw` with `"signedTxHash", signedTx.Hash()`

Reply:
> Would prefer not to have such gigantic output in log lines

Why is that? In my head a log line needs to answer two questions:
- where is this coming from? The log line needs to point to a unique place in the source.
- how do I reproduce it? The log line needs to provide all the information needed to reach that same state in a debug environment. 

I'm more concerned that we are not using structured logging. 
How do we aggregate logs? Do we use an external service? Can I get access to that?

Reply:
We do use structured logging! A potential future improvement is to have the tx hash be a tag instead of part of the message.

Reply:
Sure, done.

Reply:
If you watch the output of chainlink, most "messages" are quite short, then you have a set of key value pairs which contain additional information.

When they are all like this, the logs become much easier to read. Data wraps more cleanly.

You'll notice occasionally then a log with a huge message, and it messes up all the formatting and those values are harder to parse out because they aren't in a consistent format.